Bio/motifs/_pwm.c

```diff
@@ -46,16 +27,16 @@ calculate(const char sequence[], int s, PyObject* matrix, npy_intp m)
                  plasmid). */
                 case 'A':
                 case 'a':
-                    score += *((double*)PyArray_GETPTR2(array, j, 0)); break;
+                    score += matrix[j*4+0]; break;
                 case 'C':
                 case 'c':
-                    score += *((double*)PyArray_GETPTR2(array, j, 1)); break;
+                    score += matrix[j*4+1]; break;
                 case 'G':
                 case 'g':
-                    score += *((double*)PyArray_GETPTR2(array, j, 2)); break;
+                    score += matrix[j*4+2]; break;
                 case 'T':
                 case 't':
-                    score += *((double*)PyArray_GETPTR2(array, j, 3)); break;
+		    score += matrix[j*4+3]; break;
```

Comment:
Is there a strange tab/space change happening here?

Reply:
You're right, there were two tabs there. I made another commit to replace them by spaces.